src/runtime/components/navigation/Pagination.vue

```diff
@@ -1,12 +1,21 @@
 <template>
   <UButtonGroup :size="size">
     <UButton
-      v-if="prevIcon"
-      :icon="prevIcon"
+      v-if="prevLabel || prevIcon"
       :disabled="!canGoPrev"
       v-bind="{ ...ui.default.inactiveButton, ...inactiveButton }"
       @click="onClickPrev"
-    />
+    >
+      <slot name="prev">
```

Comment:
I would move the slot around the button with the `onClickPrev` method to allow for more customization.